### DISCUSSION TITLE
Verschiedene Gebäusegrössen?

### DISCUSSION THREAD

**Discussion by Varone3000** [R3NE07/Futro-S740#20] [Q&A]
Moin Leute,

kann es sein das es vom Futro s720 eine kleine und eine große Variante gibt?
Hab gestern eine Mini Variante gekauft und war erstaunt, das dort kein Platz für ne 2.5 Zoll SSD ist. Bei Fujitsu find ich da nicht so wirklich Hilfe. Gruß Olli

**Comment by CaneTLOTW** [comment]
Teile doch mal nen paar Bilder.

S720 ist doch aber auf AMD basis, oder nicht?
